editor/assets/stylesheets/_z-index.scss

```diff
@@ -13,8 +13,8 @@ $z-layers: (
 	'.editor-post-visibility__dialog': 30,
 	'.editor-post-schedule__dialog': 30,
 	'.editor-block-mover': 30,
-	'.components-notice-list': 100000,
-	'.components-popover': 100000,
```

Comment:
What if you open the inserter (which is a popover on a small screen), Should the inserter show up above or under the admin bar? I think it should be above